airbyte-webapp-e2e-tests/cypress.json

```diff
@@ -1,13 +1,7 @@
 {
   "projectId": "916nvw",
   "baseUrl": "http://localhost:3000",
-  "testFiles": [
-    "base.spec.js",
-    "onboarding.spec.js",
-    "connection.spec.js",
-    "destination.spec.js",
-    "source.spec.js"
-  ],
+  "testFiles": ["**/*.spec.*"],
```

Comment:
:tada: way better :) (especially since we anyway should never rely on a specific ordering of the test suites execution - which we did in the past which already broke when we switched to cypress dashboard).